daemon/main.c

```diff
@@ -96,6 +96,15 @@ void web_server_threading_selection(void) {
     }
 }
 
+int make_dns_decision(const char *config_name, const char *default_value, SIMPLE_PATTERN *p) {
+    char *value = config_get(CONFIG_SECTION_WEB,config_name,default_value);
+    if(!strcmp("yes",value))
+        return 1;
+    if(!strcmp("no",value))
+        return 0;
+    return simple_pattern_is_potential_name(p);
```

Comment:
According to the documentation this function will test three possible values "yes", "no" and "heuristic", but I am seeing here that case "yes" and "no" are no set you are moving with the default value "heuristic" that is something safe, but I will give a suggestion, please check in the function `web_server_config_option` whether the user is given an expected value and case it is not write call the function `error()` to report, because it is possible our user will write something like "yea" and he will report us bug, because we did not say anywhere that he wrote something wrong and we are assuming the default option.

Reply:
Good suggestion - fixed.